src/azure-cli/azure/cli/command_modules/cdn/tests/latest/afdx_scenario_mixin.py

```diff
@@ -45,7 +45,7 @@ def afd_profile_show_cmd(self, group, name, checks=None):
         return self.cmd(command, checks)
 
     def afd_profile_delete_cmd(self, group, name, checks=None):
-        command = 'afd profile delete -g {} --profile-name {}'.format(group, name)
+        command = 'afd profile delete -g {} --profile-name {} -y'.format(group, name)
```

Comment:
Previously, `az afd profile delete` didn't have command confirmation, please remove it in the workspace UI.